tests/testthat/test-qenv_join.R

```diff
@@ -60,15 +60,15 @@ testthat::test_that("join does not duplicate code but adds only extra code", {
   q2 <- eval_code(q2, quote(mtcars2 <- mtcars))
 
   testthat::expect_true(.check_joinable(q1, q2))
-  q <- join(q1, q2)
+  q <- c(q1, q2)
 
   testthat::expect_identical(
     q@code,
     c("iris1 <- iris\nmtcars1 <- mtcars", "iris2 <- iris", "mtcars2 <- mtcars")
   )
 
   testthat::expect_equal(
-    as.list(q@env),
+    as.list(q@.xData),
     list(iris1 = iris, iris2 = iris, mtcars1 = mtcars, mtcars2 = mtcars)
```

Comment:
I'd rather check this way - pure result vs expectation
```suggestion
    q@.xData,
    list2env(list(iris1 = iris, iris2 = iris, mtcars1 = mtcars, mtcars2 = mtcars))
```